indico/modules/events/papers/templates/forms/paper_email_settings_widget.html

```diff
@@ -0,0 +1,118 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    {% set content_reviewing_enabled = field.event.cfp.content_reviewing_enabled %}
+    {% set layout_reviewing_enabled = field.event.cfp.layout_reviewing_enabled %}
+    <input type="hidden" id="{{ field.id }}" name="{{ field.name }}" value="{{ field._value()|tojson|forceescape }}">
+    <div id="paper-email-settings-widget-{{ field.id }}">
+        <table class="i-table-widget">
+            <thead>
+                <tr>
+                    <th>{% trans %}Notify when{% endtrans %}</th>
+                    <th>{% trans %}Judge{% endtrans %}</th>
+                    {% if content_reviewing_enabled %}
+                        <th>{% trans %}Content reviewer{% endtrans %}</th>
+                    {% endif %}
+                    {% if layout_reviewing_enabled %}
+                        <th>{% trans %}Layout reviewer{% endtrans %}</th>
+                    {% endif %}
+                    <th>{% trans %}Author{% endtrans %}</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr class="multiple-recipients">
+                    <td>{% trans %}the person is added to/removed from the conference{% endtrans %}</td>
+                    <td>
+                        <input type="checkbox" name="notify_on_added_to_event" id="notify_on_added_to_event-judge"
```

Comment:
I don't think we are using the `id` for anything. I would leave it out if that's the case. Also, inconsistent underscore/hyphen. There are some more instances of the same in this file.